### DISCUSSION TITLE
FastAPI MongoDB Multiple Databases Connection - Slow to the point makes it unusable

### DISCUSSION THREAD

**Discussion by immrakpk** [fastapi/fastapi#8629] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.on_event("startup")
async def startup_db_client():
    app.mongodb_client = AsyncIOMotorClient(settings.DB_URL)
    app.mongodb = app.mongodb_client[settings.DB_NAME]
    app.auth_mongodb = app.mongodb_client[settings.AUTH_DB_NAME]

@app.on_event("shutdown")
async def shutdown_db_client():
    app.mongodb_client.close()
```

### Description

Here's the issue: I can call both mongodb and auth mongodb concurrently using motorio and pymongo, but if I call more than one db, it gets stuck and I have to wait for at least 30 to 40 seconds before my request is executed. Do you have any suggestions about how to properly use several MongoDB databases in fastapi?

Thanks guys, appreciate any help I could get.

### Operating System

Linux

### Operating System Details

Ubuntu 20 LTS

### FastAPI Version

0.70.0

### Python Version

3.9

### Additional Context

I want to deploy a semi-microservice with multiple fastapi instances that communicate with one another. This is a question about this initiative.

**Comment by raphaelauv** [comment]
It doesn't sound a fastapi issue, you can close this issue and open a discussion -> https://github.com/tiangolo/fastapi/discussions
